client/src/explorer/gatewayExplorer.ts

```diff
@@ -177,9 +178,12 @@ export class BlockchainGatewayExplorerProvider implements BlockchainExplorerProv
                     arguments: [runtimeGateway]
                 };
 
+                let gatewayName: string = runtimeGateway.name;
```

Comment:
you don't need both these lines you only need to set the gateway name

Reply:
Done